scripts/run-load-test.sh

```diff
@@ -0,0 +1,420 @@
+        echo "$1:$2:$k $pkg_vx copied" >> "$result_file"
+        create_package_revision "$pkg_vx"
+        echo "$1:$2:$k $pkg_vx approved, took $(echo $EPOCHREALTIME-$start_prvx | bc -l) seconds"  >> "$result_file"
+    done
+}
+
+create_repo () {
+    git_repo_name="porch-scale-test-$1"
+
+    echo "creating repo $git_repo_name . . ."
+    {
+        curl -k -H "content-type: application/json" "http://nephio:secret@$git_repo_server:3000/api/v1/user/repos" --data '{"name":"'"$git_repo_name"'"}' >> "$log_file" 2>&1
+
+        TMP_DIR=$(mktemp -d)
+        pushd "$TMP_DIR" || return
+        git clone "http://nephio:secret@$git_repo_server:3000/nephio/$git_repo_name.git"
+
+        pushd "$git_repo_name" || return
+        git switch -c  main
+        touch README.md
+        git add README.md
+        git config user.name nephio
+        git commit -m "first commit"
+        git push -u origin main
+        popd || return
+
+        popd || return
+
+        if ! $dirty_mode
+        then
+            rm -fr "$TMP_DIR"
+        fi
+    } >> "$log_file" 2>&1
+
+    kubectl apply -f - <<EOF >> "$log_file" 2>&1
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Repository
+
+metadata:
+   name: "$git_repo_name"
+   namespace: porch-scale
+
+spec:
+    description: "$git_repo_name"
+    content: Package
+    deployment: false
+    type: git
+    git:
+        repo: "http://$git_repo_server:3000/nephio/$git_repo_name.git"
+        directory: /
+        branch: main
+        createBranch: true
+        secretRef:
+            name: gitea
+EOF
+    
+    echo "created repo $git_repo_name"
+
+    for j in $(seq 1 "$package_count")
+    do 
+        create_package "$git_repo_name" "$j" >> "$log_file" 2>&1
+    done
+}
+
+create_namespace_secret() {
+    kubectl apply -f - <<EOF >> "$log_file" 2>&1
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: porch-scale
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gitea
+  namespace: porch-scale
+type: kubernetes.io/basic-auth
+data:
+  username: bmVwaGlv
+  password: c2VjcmV0
+EOF
+}
+
+delete_package_revisions () {
+    for pkg in $(porchctl rpkg get -n porch-scale | grep -v 'WORKSPACENAME' | cut -f1 -d' ')
+    do
+        start_prvd=$EPOCHREALTIME
+        porchctl -n porch-scale rpkg propose-delete "$pkg" >> "$log_file" 2>&1
+        porchctl -n porch-scale rpkg delete "$pkg" >> "$log_file" 2>&1
+        echo "$pkg deleted, took $(echo $EPOCHREALTIME-$start_prvd | bc -l) seconds"
+    done
+}
+
+delete_repo () {
+    git_repo_name="porch-scale-test-$1"
+
+    echo "deleting repo $git_repo_name . . ."
+    curl -k -H "content-type: application/json" --request DELETE "http://nephio:secret@$git_repo_server:3000/api/v1/repos/nephio/$git_repo_name" >> "$log_file" 2>&1
+    kubectl delete -n porch-scale repositories.config.porch.kpt.dev "$git_repo_name" >> "$log_file" 2>&1
+    echo "deleted repo $git_repo_name"
+}
+
+delete_namespace_secret() {
```

Comment:
I think we can just delete the namespace and the secret get deleted along with it.

kubectl delete ns porch-scale